### ISSUE TITLE
Where is the canonical hook for an off-thread task to be running just *after* the WSGI app has started?

### ISSUE THREAD

**Issue by jMyles** [hangarunderground/hendrix#83]
For example, let's say a user has a callable which they want to run off-thread, but only after settings.configure has run for a Django app.

It's not clear how to do this without either monkey-patching the WSGI object to place to defer the signal handler or having to subclass both the Deploy and Service objects.

Surely there's a smarter way?

**Comment by jMyles** [comment]
I ignored arguably the most obvious way: to create a Service and append it to the services list of the HendrixDeploy object.

Is this the consensus view?  If so, I'll close this wontfix.
